ExamenParcial2.java

```diff
@@ -0,0 +1,81 @@
+class ExamenParcial2 {
+    public static void main(String[] args) {
+        final String ventanaCerrada=":[ ]:";
+        final String luzApagada=":[º]:";
+        final String luzEncendida=":[*]:";
+        final String rayoCaido=":[X]:";
+        final String mantenimiento=":[#]:";
+        final int numeroPlantas=7;
+        final int numeroHabitaciones=6;
+        final int horasMaximas=24;
```

Comment:
Muy importante, como mínimo:

```suggestion
        final String VENTANA_CERRADA = ":[ ]:";
        final String LUZ_APAGADA = ":[º]:";
        final String LUZ_ENCENDIDA = ":[*]:";
        final String RAYO_CAIDO = ":[X]:";
        final String EN_MANTENIMIENTO = ":[#]:";
        final int NUMERO_PLANTAS = 7;
        final int NUMERO_HABITACIONES = 6;
        final int HORAS_MAXIMAS = 24;
```

Y ya mejor aun:

```suggestion
        final String WINDOW_CLOSED = ":[ ]:";
        final String WINDOW_OFF = ":[º]:";
        final String WINDOW_ON = ":[*]:";
        final String WINDOW_DAMAGED = ":[X]:";
        final String WINDOW_UNDER_MAINTEANCE = ":[#]:";
        final int FLOORS = 7;
        final int ROOMS = 6;
        final int HOURS_BY_DAY = 24;
```